app/(app)/explore/users/page.tsx

```diff
@@ -7,28 +7,35 @@ import useSWR from 'swr'
 import Text from '@/components/Text'
 import Link from 'next/link'
 import Image from 'next/image'
-import VerifyIcon from '@/components/Icon/Verify'
 import React, { useMemo, useState } from 'react'
 import { APIParams } from '@/services/api/types'
 import { Pagination, Spinner } from 'flowbite-react'
 import { getUserAvatarImage, getUserCoverImage } from '@/utils/string'
-import VerifyDisableIcon from '@/components/Icon/VerifyDisable'
-import Icon from '@/components/Icon'
+import UserFollow from "@/components/Pages/MarketplaceNFT/UserDetails/UserFollow";
+import { formatDisplayedNumber } from "@/utils";
+import useAuthStore from "@/store/auth/store";
+import Icon from "@/components/Icon";
+
 
 export default function ExploreUsersPage() {
   const api = useMarketplaceApi()
   const { queryString } = useUIStore(state => state)
   const { searchKey } = useExploreSectionFilters()
+  const myId = useAuthStore(state => state.profile?.id)
+
 
   const [activePagination, setActivePagination] = useState<APIParams.FetchUsers>({
     page: 1,
     limit: 20
   })
 
   const { data: users, isLoading, error } = useSWR(
-    !!queryString[searchKey] ? { ...activePagination, search: queryString[searchKey] , page: 1 } : {...activePagination, search: queryString[searchKey] },
-    params => api.fetchUsers(params),
-    { refreshInterval: 10000 }
+      !!queryString[searchKey] ? { ...activePagination, search: queryString[searchKey], page: 1 } : {
+        ...activePagination,
+        search: queryString[searchKey]
+      },
+      params => api.fetchUsers(params),
+      { refreshInterval: 100 }
```

Comment:
Refresh interval chỉ nên để thấp nhất 5s, nếu không sẽ gây network spam làm ảnh hưởng đến performance của BE server

Trong trường hợp cần refresh lại data khi thực hiện update có thể tham khảo tính năng này https://swr.vercel.app/docs/mutation#useswrmutation-basic-usage